robottelo/ui/navigator.py

```diff
@@ -43,6 +43,7 @@ def menu_click(self, top_menu_locator, sub_menu_locator,
                 "arguments[0].click();",
                 tertiary_element,
             )
+        self.wait_for_ajax()
```

Comment:
Bugfix. Without this line, if you try to navigate between 2 pages with searchbox and immediately use search like this:
```python
self.navigator.go_to_red_hat_subscriptions()
# some steps here
self.navigator.go_to_activation_keys()
self.assertIsNotNone(self.activationkey.search_key(name))
```
you'll receive ```Element is no longer attached to the DOM``` error. The root cause is webdriver trying to work with searchbox from page #1 (as page #2 wasn't loaded yet) and then fails when page #2 overrides it.